services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala

```diff
@@ -29,8 +30,8 @@ final case class WriteMetadataActor(batchRate: Int, flushRate: FiniteDuration)
   when(WaitingToWrite) {
     case Event(PutMetadataAction(events), curData) =>
       curData.addEvents(events) match {
-        case data@HasEvents(e) if e.length > batchRate => goto(WritingToDb) using data
-        case e => stay using e
+        case newData: HasEvents if newData.length > batchRate => goto(WritingToDb) using newData
```

Comment:
The second `e` was never short for `events`. It was short for `WriteMetadataActorData`.
The switcheroo was part to make it easier for me to follow for my debugging. I still think it's a little neater so I left it in, but its functionally identical. (shrug)

I think you do have a point though, `WriteMetadataActorData.length` should probably become `WriteMetadataActorData.eventQueueLength`, to make it even easier to follow.